etl/src/main/java/org/pih/hivmigration/etl/sql/DeIdentifyMigrator.groovy

```diff
@@ -158,6 +158,64 @@ class DeIdentifyMigrator extends SqlMigrator {
                     WHEN p.source_infant_id % 19 = 18 THEN 'Lucas'
                 END)
             where given_name != 'UNKNOWN' and pn.person_id = p.person_id and (p.gender != 'F' or p.gender is null);
+            
+            
+            update person_name pn, person p
```

Comment:
Can we move all this up under the person name de-identification, instead of being after the infant name de-identification?